### PR TITLE
Feat(VETS-CPC-335): Dockerfiles Error Fix For All Services

### PR DESCRIPTION
**Pull request by Louenes099**
Sorry Christine I was too lazy to commit after each service but the fix is there enjoy everyone!

### CODE REVIEW

**Review by VladPopa616 — APPROVED** [review]

**Review by cjayneb — APPROVED** [review]
LGTM

File: billing-service/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+FROM gradle:6.9.0 as builder
 WORKDIR /usr/src/app
 COPY src ./src
 COPY build.gradle .
 RUN ["gradle", "bootJar"]
 
 EXPOSE 8080
 
-FROM openjdk:11
+FROM openjdk:8-alpine
```
Comment (Fube) [inline billing-service/Dockerfile:9]: Are we sure billing wants to use JDK 8?
Maybe they left it as JDK 11 because they rely on JDK 11 specific features?

Comment (Louenes099) [reply billing-service/Dockerfile:9]: okay should i change it for eveyone?

**Comment by Louenes099** [comment]
Changing to jdk 8 for the billing service fixed their failing service

**Pull request merged**